sass/directives/02_base_components/boxes/_boxes.scss

```diff
@@ -90,6 +90,21 @@
 
 // Item Bar
 
+$_item-bar-breakpoint: 750px;
```

Comment:
:smiley: :smiley: :smiley: :smiley: :smiley: :smiley: :smiley: :smiley: :smiley:

Airbnb actually uses that when the variable is specific to the file. I tried that but there were still times when I needed the variable in both Style Base and Employer